File: Problems_on_Arrays/FrequencyArray.java
package Problems_on_Arrays;
import java.util.*;

public class FrequencyArray {
        public static void main(String[] args) {
        Scanner sc=new Scanner(System.in);
        System.out.println("enter the number of elements:");
        int n=sc.nextInt();
        int[] arr=new int[n];
        HashMap<Integer,Integer> map=new HashMap<>();
        for (int i=0;i<n;i++){
            arr[i]=sc.nextInt();
            map.put(arr[i],map.getOrDefault(arr[i],0)+1);
        }
        for (int i:map.keySet()){
            System.out.println("the frequency of "+i+" in array is "+map.get(i));
        }

        sc.close();
    } 
}

File: Problems_on_Arrays/FrequencyArray.py
n=int(input("Enter number of elements: "))
arr=list(map(int,input("Enter elements: ").split()))

freq={}

for i in arr:
    if i in freq:
        freq[i]=freq[i]+1
    else:
        freq[i]=1

print("Frequency of elements:")
for key,value in freq.items():
    print("Frequency of ",key,"is",value)
    